build.gradle

```diff
@@ -145,6 +145,7 @@ docker {
             into '.'
         }
     }
+    noCache true
```

Comment:
Is it nice to have in other containers ?

Reply:
Probably yes, it works to get the latest packages with `apt-get` or `yum` at the build time.